### DISCUSSION TITLE
Is there a publisher app?

### DISCUSSION THREAD

**Discussion by colemickens** [scottlamb/retina#39] [Ideas]
Hi, I love what I'm seeing here and in Moonfire DVR (and I found the Coral playground stuff too). Very exciting.

I'm curious, with as much as you have here already, do you have a tiny publisher application? I'd love to see a tiny Rust rtsp client able to run on cheap, even older RPi hardware.

**Comment by scottlamb** [comment]
What do you mean by publisher?

**Comment by colemickens** [comment]
Sorry, what I mean is basically a Rust version of `v4l2rtspserver` or `rstp-simple-server`, powered by `retina`. (I might be under-estimating how much effort that sort of tool itself takes.)

**Comment by scottlamb** [comment]
Ahh. I'd like to have that, but no, not yet. There aren't even the library parts one would need to serve anything. PRs welcome if you're interested in working on this!
